include/dfrn.php

```diff
@@ -1906,12 +1914,13 @@ private static function update_content($current, $item, $importer, $entrytype) {
 	 * @param array $item the new item record
 	 *
 	 * @return int Is it a toplevel entry, a comment or a relayed comment?
+	 * @todo set proper type-hints (array?)
```

Comment:
Please see above, it can be done with PHP 5 (PHP 7 supports native type-hints, btw.)